x509-cert/src/time.rs

```diff
@@ -62,7 +62,7 @@ impl Time {
 }
 
 impl fmt::Display for Time {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> core::result::Result<(), fmt::Error> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
```

Comment:
nit:
```suggestion
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```
https://doc.rust-lang.org/stable/core/fmt/type.Result.html

Reply:
Fixed.